packages/google_workspace/data_stream/rules/fields/fields.yml

```
@@ -0,0 +1,120 @@
+- name: google_workspace.rules
+  type: group
+  fields:
+    - name: actions
+      type: keyword
+      description: List of actions taken. For a list of possible values refer to https://developers.google.com/admin-sdk/reports/v1/appendix/activity/rules.
```

```suggestion
      description: List of actions taken. For a list of possible values refer to `actions` in the [event details table]{https://developers.google.com/admin-sdk/reports/v1/appendix/activity/rules#rule_match).
```
I'd recommend similar changes for the fields below and in package-field.yml where there is a reference to the API docs as well.

---

Please take a look again at this and below.

---

Hey @efd6, I have already updated this.
https://github.com/elastic/integrations/blob/9eba0eb90f1c3882f2eabf2866fcfc4ba027ebfe/packages/google_workspace/data_stream/rules/fields/fields.yml#L6
Pleases let me know if their are any other changes.

---

Oh! Sorry, UI/brain failure; the link goes to the original which doesn't have the change. Apologies.